apps/dashboard/src/components/workflow-editor/steps/shared/configure-preview-accordion.tsx

```diff
@@ -0,0 +1,79 @@
+import { CSSProperties, useEffect, useRef, useState } from 'react';
+import { Code2 } from '@/components/icons/code-2';
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from '@/components/primitives/accordion';
+import { Button } from '@/components/primitives/button';
+import { Editor } from '@/components/primitives/editor';
+import { loadLanguage } from '@uiw/codemirror-extensions-langs';
+
+const extensions = [loadLanguage('json')?.extension ?? []];
+
+interface ConfigurePreviewAccordionProps {
```

Comment:
As discussed with the team, we should prefer types.

Reply:
Old habbits... need to eslint this one